lib/nylas/resources/webhooks.rb

```diff
@@ -33,9 +33,9 @@ class Webhooks < Resource
 
     # Return all webhooks.
     #
-    # @return [Array(Array(Hash), String)] The list of webhooks and API Request ID.
+    # @return [Array(Array(Hash), String, String)] The list of webhooks, API Request ID, and next cursor.
     def list
-      get(
+      get_list(
```

Comment:
Same comment as the grants endpoint - I believe our UAS endpoints uses limt/offset pagination